src/hexdoc/patchouli/flag.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any
+
+from pydantic import model_validator
+
+from hexdoc.model import HexdocModel
+
+
+class Flag(HexdocModel):
+    name: str
+    negated: bool = False
+
+    @model_validator(mode="before")
+    @classmethod
+    def parse_flag(cls, data: Any) -> Any:
+        if isinstance(data, str):
+            assert (
+                "," not in data
+            )  # not sure if there are other invalid characters or not
```

Comment:
* what's this for?
* move the comment to the line above and it'll be formatted a bit less weirdly
* this should have some sort of useful error message, not just an empty assert (ie. `assert condition, "message goes here"`)